rocAL/rocAL/include/meta_data/meta_data.h

```diff
@@ -119,6 +136,15 @@ struct BoundingBox : public MetaData
         _bb_label_ids = std::move(bb_label_ids);
         _img_size = std::move(img_size);
     }
+    BoundingBox(BoundingBoxCords bb_cords,BoundingBoxLabels bb_label_ids ,ImgSize img_size, MaskCords mask_cords, std::vector<int> polygon_count, std::vector<std::vector<int>> vertices_count)
```

Comment:
Proper alignment required for the comma in this line please check

Reply:
Its valid to pass arguments by value for std::vove